### DISCUSSION TITLE
[Kavita] Sequel / Prequel should be a Two Way Relationship.

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2551] [Ideas]
### Idea Description

Posted by XyberDAWG
· 12 months ago
· 15 Upvotes
https://feats.kavitareader.com/posts/276/sequel-prequel-should-be-a-two-way-relationship

Description

It seems to me, when ever I enter a "Sequel" then have to go to the sequel Series I have to enter "Prequel" for the relationship. This seems like it should be an automatic, as I don't know of any cases where a Prequel or Sequel would not go both directions. I was going to enter it as a bug, but I think its just not set up that way.

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [x] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
This has been delivered in v0.8.2.9 and will be available in v0.8.3.
